app/src/main/java/de/tum/in/tumcampusapp/component/tumui/grades/ExamListAdapter.java

```diff
@@ -21,24 +22,24 @@
  */
 public class ExamListAdapter extends SimpleStickyListHeadersAdapter<Exam> {
     private static final DateFormat DF = DateFormat.getDateInstance(DateFormat.MEDIUM);
-    private static final Map<String, Integer> GRADE_DRAWABLE = ImmutableMap.<String, Integer>builder()
-            .put("1,0", R.drawable.grade_1_0)
-            .put("1,3", R.drawable.grade_1_3)
-            .put("1,4", R.drawable.grade_1_3)
-            .put("1,7", R.drawable.grade_1_7)
-            .put("2,0", R.drawable.grade_2_0)
-            .put("2,3", R.drawable.grade_2_3)
-            .put("2,4", R.drawable.grade_2_3)
-            .put("2,7", R.drawable.grade_2_7)
-            .put("3,0", R.drawable.grade_3_0)
-            .put("3,3", R.drawable.grade_3_3)
-            .put("3,4", R.drawable.grade_3_3)
-            .put("3,7", R.drawable.grade_3_7)
-            .put("4,0", R.drawable.grade_4_0)
-            .put("4,3", R.drawable.grade_4_3)
-            .put("4,4", R.drawable.grade_4_3)
-            .put("4,7", R.drawable.grade_4_7)
-            .put("5,0", R.drawable.grade_5_0)
+    private static final Map<String, Integer> GRADE_COLOR = ImmutableMap.<String, Integer>builder()
```

Comment:
Can we move this out of the ExamListAdapter and into Grade itself, e.g. exposing it via grade.getGradeColor(context)? Would help simplify the getView() method.

Reply:
I like that suggestion - @jacqueline8711 could you quickly switch this up please?